frontend/src/context/TypeAndAuthProvider.js

```diff
@@ -11,19 +12,23 @@ export default function TypeAndAuthProvider({children}){
     const WEEK = "week"
```

Comment:
I would move those to an extra file and simply export them. In that way you can use them without calling the custom hook.